definitions/nodes.md

```diff
@@ -1,13 +1,20 @@
 ---
 title: Nodes
 status: Feedback Appreciated
-category: concept
+category: Concept
 ---
+
 ## Nodes
 
-A node is a computer that works in concert with other computers, or nodes, to accomplish a common task. When we talk about cloud native platforms a node could be a physical computer, a virtual computer, referred to as a VM, or even a container.
+### What it is
+
+A node is a computer that works in concert with other computers, or nodes, to accomplish a common task. When we talk about cloud native platforms a node could be a physical computer, a virtual computer, referred to as a [VM](virtual_machine.md), or even a [container](container.md).
```

Comment:
A node doesn't have to be a computer. Here's my suggested change. I also included an example:

A node is an independent compute element that is connected with other nodes over a network forming a [cluster](cluster.md). Nodes can be multi-functional physical or virtual machines, or perform a very specific function within the network. Take your laptop, modem, and printer, for example. They are all connected over your wifi network communicating and collaborating, each representing one node. In [cloud computing](cloud_computing.md), nodes generally refer to [VMs](virtual_machine.md) or [containers](container.md) that accomplish a common goal. 

@JasonMorgan not sure if "compute element" is the right word

Reply:
Or distinct unit of compute.